sig-contributor-experience/annual-report-2021.md

```diff
@@ -4,79 +4,105 @@
-2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
--->
-
-   - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-   - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-   - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+Contribex is a service and program orientated SIG. Most of our initiatives cover long term services for the Kubernetes project.
+
+
+  |         Subproject        |          Initiative / Program          |
+  |:-------------------------:|:--------------------------------------:|
+  | Community                 | Community Repo Stewardship             |
+  | Community Management      | Annual Contributor Survey              |
+  | Community Management      | Calendar Admin                         |
+  | Community Management      | Chair and TL Meetings + Docs           |
+  | Community Management      | Discuss Admin                          |
+  | Community Management      | Mailing List Admin                     |
+  | Community Management      | Slack Admin                            |
+  | Community Management      | Zoom / YouTube Admin                   |
+  | Contributor Documentation | Contributor Guide Stewardship          |
+  | Contributor Documentation | Contributor Site                       |
+  | Contributor Documentation | Developer Guide Audit                  |
+  | Contributor Documentation | Developer Guide Stewardship            |
+  | Contributors Comms        | Contributor / SIG Profiling            |
+  | Contributors Comms        | Stewardship of k8scontributors twitter |
+  | Devstats                  | Devstats Dashboard Update              |
+  | Events                    | Monthly Community Meeting              |
+  | Events                    | Office Hours                           |
+  | Events                    | Elections                              |
+  | GitHub Management         | GitHub Admin / Moderation              |
+  | GitHub Management         | GitHub Master -> Main rename           |
+  | GitHub Management         | GitHub New Membership Coordinator      |
+  | Mentoring                 | Google Summer of Code                  |
+  | Mentoring                 | Group Mentoring                        |
+  | Mentoring                 | LFX Mentor Program                     |
+  | Mentoring                 | Meet our Contributors                  |
+  | Mentoring                 | Outreachy                              |
```

Comment:
```suggestion
  | Mentoring                 | Outreachy                              |
  | Mentoring                 | New Contributor Workshop  |
```

Reply:
We didn't really do anything with the NCW last year

Reply:
I read this as things we are working on currently

Reply:
it would be cool if this list had issues and things attached to it like what it is we are working on in each area and maybe a vision statement for each. I can help with this.

Reply:
added! Thank you @mrbobbytables for putting the table together 🙏